SonarQube.Common/AnalysisConfig/AnalysisConfigExtensions.cs

```diff
@@ -18,9 +19,9 @@ public static class ConfigSettingsExtensions
         #region Public methods
 
         /// <summary>
-        /// Returns the value of the specified setting, or the supplied default value if the setting could not be found
+        /// Returns the value of the specified config setting, or the supplied default value if the setting could not be found
         /// </summary>
-        public static string GetSetting(this AnalysisConfig config, string settingId, string defaultValue)
+        public static string GetConfigValue(this AnalysisConfig config, string settingId, string defaultValue)
```

Comment:
I've renamed some of the methods to try to make it clearer which ones are dealing with general configuration settings and which ones are dealing with analysis settings.